azure-quantum/tests/unit/test_qiskit.py

```diff
@@ -515,6 +515,28 @@ def test_translate_quantinuum_qir(self):
         assert "targetCapability" in params
 
 
+    @pytest.mark.quantinuum
```

Comment:
```suggestion
    @pytest.mark.quantinuum
    @pytest.mark.live_test
```

I believe this is missing the `live_test` mark